packages/jasmine-runner/README.md

```diff
@@ -1,11 +1,11 @@
-[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fstryker-mutator%2Fstryker%2Fmaster%3Fmodule%3Djasmine-runner)](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker/master?module=jasmine-runner)
-[![Build Status](https://github.com/stryker-mutator/stryker/workflows/CI/badge.svg)](https://github.com/stryker-mutator/stryker/actions?query=workflow%3ACI+branch%3Amaster)
+[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fstryker-mutator%2Fstryker%2Fmaster%3Fmodule%3Djasmine-runner)](https://dashboard.stryker-mutator.io/reports/github.com/stryker-mutator/stryker-js/master?module=jasmine-runner)
+[![Build Status](https://github.com/stryker-mutator/stryker-js/workflows/CI/badge.svg)](https://github.com/stryker-mutator/stryker-js/actions?query=workflow%3ACI+branch%3Amaster)
 [![NPM](https://img.shields.io/npm/dm/@stryker-mutator/jasmine-runner.svg)](https://www.npmjs.com/package/@stryker-mutator/jasmine-runner)
 [![Node version](https://img.shields.io/node/v/@stryker-mutator/jasmine-runner.svg)](https://img.shields.io/node/v/@stryker-mutator/jasmine-runner.svg)
 [![Slack Chat](https://img.shields.io/badge/slack-chat-brightgreen.svg?logo=slack)](https://join.slack.com/t/stryker-mutator/shared_invite/enQtOTUyMTYyNTg1NDQ0LTU4ODNmZDlmN2I3MmEyMTVhYjZlYmJkOThlNTY3NTM1M2QxYmM5YTM3ODQxYmJjY2YyYzllM2RkMmM1NjNjZjM)
 
-# Stryker Jasmine Runner
+# StrykerJS Jasmine Runner
```

Comment:
Perhaps just 'Jasmine Runner'?

Reply:
(Same goes for the other packages)

Reply:
It isn't the jasmine test runner itself, right?

Reply:
hmm fair enough